tests/lib/prepare-project.sh

```diff
@@ -32,6 +44,28 @@ build_deb(){
     cp ../*.deb "$GOHOME"
 }
 
+fedora_build_rpm() {
```

Comment:
If we want to, we could use `rpmbuild` for both distros. We'd need to use `dnf builddep` for Fedora to install the builddeps and whatnot, but it's doable. It gets tricky due to the switches we flip, though.